element_miniscope/miniscope.py

```diff
@@ -75,20 +60,10 @@ def activate(
 
 
 def get_miniscope_root_data_dir() -> list:
-    """
-    All data paths, directories in DataJoint Elements are recommended to be stored as
-    relative paths (posix format), with respect to some user-configured "root"
-    directory, which varies from machine to machine (e.g. different mounted drive
-    locations).
-
-    get_miniscope_root_data_dir() -> list
-        This user-provided function retrieves the possible root data directories
-         containing the miniscope data for all subjects and sessions
-         (e.g. acquired Miniscope-DAQ-V4 raw files,
-         output files from processing routines, etc.).
-
-        :return: a string for full path to the root data directory,
-         or list of strings for possible root data directories
+    """Pulls relevant func from parent namespace to specify root data dir(s).
```

Comment:
Please adjust to 
1. ensure no info is lost in this conversion.
2. have a oneliner, followed by a more descriptive pgf. 
3. add a `Returns` section

Reply:
Done